src/pages/studentLogin/StudentLogin.jsx

```diff
@@ -63,13 +69,14 @@ const StudentLogin = ({ type }) => {
   };
   useEffect(() => {
     axios
-      .get(`${API_END_POINT}/api/task/${batchId}/list/user/task/?filter_task_type=${type === "assessment" ? 1 : 0 }`, { headers })
+      .get(`${API_END_POINT}/api/task/${batchId}/list/user/task/`, { headers })
```

Comment:
why this line has been removed functionality is breaking,

Reply:
Now All is corrected.